src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -365,11 +376,11 @@ public bool IsMethodGroup
         /// <summary>
         /// Returns true if the conversion is an implicit tuple conversion.
         /// </summary>
-        public bool IsTuple
+        public bool IsTupleLiteral
```

Comment:
Probably. They are not used much internally and we need to think about right names since this is a public API. I will enter an issue on these for now.